database/suggestions.js

```diff
@@ -111,7 +114,8 @@ module.exports.getSuggestionsWithCountedVotes = async (episode = {}) => {
             voteCount,
         };
     });
-    return formattedCountedSuggestions;
+    if (getEpNum) return [episode.epNum, formattedCountedSuggestions];
+    else return formattedCountedSuggestions;
```

Comment:
Seems like you actually need two functions. I understand the separation is not easy though.